test/sh/invoke.sh

```diff
@@ -15,4 +15,4 @@
 
 # This is script is for programmatic invocation of a shell script
 echo "$(date +"%Y-%m-%d %H:%M:%S")" -- invoking script $* >>/tmp/invoke.log
-nohup $* &
+nohup sudo $* &
```

Comment:
Normally this is done y developer who has root access, not sure sudo is required here

Reply:
Hi @alikdell,
We're currently passing sudo when executing the invoke.sh script, but not when running the bounce.ip file thorugh the invoke.sh. As a result, the systemctl commands inside bounce.ip are failing, since they require permissions as we're operating as the core user in the OCP environment.
Thank You.!